twoliter/embedded/build.Dockerfile

```diff
@@ -110,10 +115,15 @@ COPY ./packages/${PACKAGE}/ .
 
 COPY --chown=builder --from=rpm-macros-and-bconds /home/builder/generated.* .
 
-# Merge generated bconds with the package spec, and put sources in the right place.
+# Merge generated bconds with the package spec, and put sources in the right place. Set Epoch values
+# for the package and all of its subpackages. Additionally, for packages declaring explicit package versions
+# via "{Requires,Conflicts,Obsoletes}: = ...", ensure the epoch for the specified package is set.
 RUN \
    cat "/usr/lib/rpm/platform/${ARCH}-bottlerocket/macros" generated.rpmmacros > .rpmmacros \
+   && echo -e -n "Epoch: ${BUILD_EPOCH}\n" >> rpmbuild/SPECS/${PACKAGE}.spec \
    && cat generated.bconds ${PACKAGE}.spec >> rpmbuild/SPECS/${PACKAGE}.spec \
+   && sed -i "/%package.*$/a Epoch: ${BUILD_EPOCH}" rpmbuild/SPECS/${PACKAGE}.spec \
+   && sed -i "s;\([Requires|Conflicts|Obsoletes]:.*=\) \(\%{version}\-\%{release}$\);\1 ${BUILD_EPOCH}:\2;" rpmbuild/SPECS/${PACKAGE}.spec \
```

Comment:
Why do we need this? It feels like it will be brittle in the face of our more complex boolean dependencies.

Reply:
This is needed because when we set an Epoch for a package, *and* all subpacakges within a package, we must now reference those subpackages with their Epoch, else rpmbuild will complain:

```
nothing provides bottlerocket-kernel-5.15-devel = 5.15.158-11717197220.a87879e6.br1 needed
 by bottlerocket-kernel-5.15-1:5.15.158-11717197220.a87879e6.br1.aarch64 from repo
```

We only have these situations in kernel package in b-os/b:

```
$   git grep '%{version}' | grep 'Requires'
packages/kernel-5.10/kernel-5.10.spec:Requires: %{name}-modules = %{version}-%{release}
packages/kernel-5.10/kernel-5.10.spec:Requires: %{name}-devel = %{version}-%{release}
packages/kernel-5.15/kernel-5.15.spec:Requires: %{name}-modules = %{version}-%{release}
packages/kernel-5.15/kernel-5.15.spec:Requires: %{name}-devel = %{version}-%{release}
packages/kernel-6.1/kernel-6.1.spec:Requires: %{name}-modules = %{version}-%{release}
packages/kernel-6.1/kernel-6.1.spec:Requires: %{name}-devel = %{version}-%{release}
```

and they all use simple `=` comparison. I agree it is definitely more robust to cover all the operators possible here, I can give that a shot